libphal/phal_dump.C

```diff
@@ -317,6 +320,11 @@ void collectSBEDump(uint32_t id, uint32_t failingUnit,
 	try {
 		// Collect SBE local register dump
 		std::vector<SBESCOMRegValue_t> sbeScomRegValue;
+		if (forOdySsey)
```

Comment:
why we need it here? it can be done in the starting right?

Reply:
Changed as suggested